tests/unit/test_utils.py

```diff
@@ -1028,9 +1028,9 @@ def test_import_member(root_logger):
 
 def test_import_member_no_such_module(root_logger):
     with pytest.raises(
-            SystemExit,
-            match=rf"Failed to import the 'tmt\.steps\.nope_does_not_exist'"
-                  rf" module from '{Path.cwd()}'."):
+        SystemExit,
+        match=rf"Failed to import the 'tmt\.steps\.nope_does_not_exist'"
+            rf" module from '{Path.cwd()}'."):
```

Comment:
Seems like an unrelated change. A valid one, but unrelated, or is it caused by a linter reformating the file?

Reply:
Aaaha, got it. I didn't suspect autopep8 to cause this, but it's true some linters do behave like this.